webapp/channels/src/components/with_tooltip/create_tooltip.tsx

```diff
@@ -15,7 +15,7 @@ type EmojiStyle = 'inline' | 'large' | undefined;
 
 export type CommonTooltipProps = {
     id: string;
-    title: string | MessageDescriptor | React.ReactElement;
+    title: string | MessageDescriptor | React.ReactElement | React.ReactNode;
```

Comment:
Why add this as an option? I always forget the exact difference between `ReactElement` and `ReactNode`, but I think it was intentional that we didn't allow `ReactNode` here